### ISSUE TITLE
Compilation problems

### ISSUE THREAD

**Issue by tzaeschke** [netsec-ethz/bootstrapper#16]
I tried to follow the installation instructions [here](https://github.com/netsec-ethz/bootstrapper/), section "Building": 

1) `make bootstrapper` fails with `make: *** No rule to make target 'bootstrapper'.  Stop.`
2) `bazel build //:bootstrapper`, `bazel build ...` fail with 
```
ERROR: /home/ztilmann/.cache/bazel/_bazel_ztilmann/87be20ea61cc62702d6eca3dd1110526/external/io_bazel_rules_go/BUILD.bazel:71:16: in (an implicit dependency) attribute of go_context_data rule @@io_bazel_rules_go//:go_context_data: in $whitelist_function_transition attribute of go_context_data rule @@io_bazel_rules_go//:go_context_data: package group '@@bazel_tools//tools/whitelists/function_transition_whitelist:function_transition_whitelist' is misplaced here (they are only allowed in the visibility attribute)
```
I am on Ubuntu 22.04.4, bazel 7.0.2, go 1.22.0, AMD64.

### Proposed workaround
Probnlem 2): Using `bazel-5.1.0` I got `bazel-5.1.0 build //:bootstrapper` to work but everything else still failed

### Proposed fix

Problem 2) disappeared when I changed the `io_bazel_rules_go` dependency in `WORKSPACE` to
```
http_archive(
    name = "io_bazel_rules_go",
    sha256 = "80a98277ad1311dacd837f9b16db62887702e9f1d1c4c9f796d0121a46c8e184",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.46.0/rules_go-v0.46.0.zip",
        "https://github.com/bazelbuild/rules_go/releases/download/v0.46.0/rules_go-v0.46.0.zip",
    ],
)
```

**Comment by FR4NK-W** [comment]
Fixed by PR #27.
Confirmed in https://github.com/netsec-ethz/bootstrapper/pull/27#issuecomment-2504245764

**Comment by dumin199101** [comment]
Nice
